datadog/dogstatsd/base.py

```diff
@@ -32,6 +32,14 @@
 # Tag name of entity_id
 ENTITY_ID_TAG_NAME = "dd.internal.entity_id"
```

Comment:
We could remove this variable and hardcode the value in the `DD_ENV_TAGS_MAPPING` dict. It's not used anywhere else I think

Reply:
Yes but @jirikuncar asked me to keep it here just in case folks are importing it and using it.

Reply:
We should not do unnecessary backward incompatible changes.

Reply:
True, I don't think anybody is using it but, just in case, let's keep it.